src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/EvalResultDataItem.cs

```diff
@@ -21,7 +21,8 @@ internal enum ExpansionKind
         RawView,
         ResultsView,
         StaticMembers,
-        TypeVariable
+        TypeVariable,
+        ExplicitEvaluationResult,
```

Comment:
Also, do we really need this Kind, or could we just null check ExternalEvaluationResult?  In the new code, we're doing a mix of both (kind check and null check)